scope/src/shared/logging.rs

```diff
@@ -84,11 +84,12 @@ impl LoggingOpts {
 
         let file_path = PathBuf::from(&full_file_name);
         let (non_blocking, guard) = tracing_appender::non_blocking(
-            File::create(file_path).expect("to be able to create log file"),
+            strip_ansi_escapes::Writer::new(File::create(file_path).expect("to be able to create log file")),
```

Comment:
@ethankhall just to confirm, this only removes color codes from the logged output, and not from the output passed through to the terminal, right?

Reply:
correct
<img width="1064" alt="image" src="https://github.com/oscope-dev/scope/assets/794802/142e5630-19fd-4262-b579-0ec501c620ed">